RiotNSE/NotificationService.swift

```diff
@@ -73,15 +77,17 @@ class NotificationService: UNNotificationServiceExtension {
     }
     
     func setupLogger() {
-        if isatty(STDERR_FILENO) == 0 {
-            MXLogger.setSubLogName("nse")
-            MXLogger.redirectNSLog(toFiles: true, numberOfFiles: 100)
+        if !NotificationService.isLoggerInitialized {
+            if isatty(STDERR_FILENO) == 0 {
+                MXLogger.setSubLogName("nse")
+                MXLogger.redirectNSLog(toFiles: true)
```

Comment:
This is fine if there is a single process for the extension. If another extension process instance can exist, it will rotate log files :/
Test and see :)

Reply:
It seems only one process for quite amount of time. Seems fine.